converter/excelHandler.js

```diff
@@ -32,14 +32,18 @@ export const parseDmnContent = ({
       aggregation,
       amountOutputs,
       tableName,
-      hitPolicy
+      hitPolicy,
+      hasAnnotationColumn
     } = sheetOptions;
 
     const rawInputData = header.slice(0, header.length - amountOutputs);
     const rawOutputData = header.slice(header.length - amountOutputs);
     const safeRuleRows = validateRows(sheet.data.slice(1));
+
     const typeRefs = getTypeRefs(safeRuleRows[0]);
 
+    console.log(typeRefs);
```

Comment:
```suggestion
```

Let's remove this one so we do not accidentally log stuff in production.

Reply:
sure, my mistake 
[649c3e8](https://github.com/pinussilvestrus/camunda-modeler-excel-import-plugin/pull/109/commits/649c3e8730b4171d92714db46d9fb463e868b061)